adapters/oidc/adapter-core/src/test/java/org/keycloak/adapters/KeycloakConfigTest.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * 
+ */
+package org.keycloak.adapters;
+
+import static org.junit.Assert.*;
+
+import org.jboss.logging.Logger;
+import org.junit.Test;
+import org.keycloak.adapters.jaas.AbstractKeycloakLoginModule;
+
+
+/**
+ * @author <a href="mailto:sarai@redhat.com">Saurabh Rai</a>
+ *
+ */
+public class KeycloakConfigTest {
```

Comment:
We don't use unit tests. Please look at testsuite for how we do proper testing

Reply:
@stianst  I am not familiar with Keycloack testsuites any help or guidance would be appreciated.  The change is just to check if the keycloakConfigFile starts with profile only then it should load it via URL handler else it will throw exception. I have updated the changes accordingly and removed this Unit test class.